less/footer.less

```diff
@@ -1,12 +1,23 @@
 .footer {
-  width: 100%;
+  width: 100vw;
+  max-width: inherit;
+  margin-top: -30px;
+  position: absolute;
   border-top: 2px dashed silver;
```

Comment:
The footer starts to have some space to its right as it approaches your `@desktop` media query. To fix that, you could either center your footer horizontally or remove `max-width: inherit` (since that setting limited the extent to which your footer could extend its width).